CMakeLists.txt

```diff
@@ -0,0 +1,98 @@
+cmake_minimum_required(VERSION 3.15)
+cmake_policy(SET CMP0091 NEW) # Use MSVC_RUNTIME_LIBRARY to select the runtime
+project(Verilator
+    VERSION 4.025.99
```

Comment:
Wrong number.   I believe this build process is still requiring autoconf, so rename this file CMakeLists.txt.in and have configure fill it in.

Reply:
I'd like to avoid using autoconf since its not windows friendly
cmake support added using this PR works without the need of any linux tool

Reply:
The runtime cmake files are currently made using autoconf, so I don't understand how that is possible.